Resources/Prototypes/Recipes/Lathes/misc.yml

```diff
@@ -223,3 +223,32 @@
   materials:
     Glass: 100
     Plastic: 100
+
+- type: latheRecipe
+  id: SpesoOne
+  result: SpaceCash
+  completetime: 1
+  materials:
+    Durathread: 0.05
```

Comment:
please double and triple check these prices do not generate infinite money and that the price is consistent (you can bulk print a single recipe, so make sure the 1 speso isn’t cheaper than a higher order by just bulk printing it)

Reply:
i literally just multiplied the base 0.05 cost by the amount of cash being made so its just qol of printing, so that ahould be fine, and its MEANT to make infinite money thru textile crate as a tradeoff, sec will arrest you if they find it but its pretty good moneymaker, plus cap will get suspicious hearing the fifteenth textile crate so you cant spam buy